lmdeploy/turbomind/deploy/target_model/base.py

```diff
@@ -333,12 +333,16 @@ def pad_weight(tensor):
 
         if emb is not None:
             emb = pad_weight(emb)
-            self.export_weight(emb, 'tok_embeddings.weight')
+            # try split along hidden dim
+            if emb.shape[1] % self.cfg.tensor_para_size == 0:
```

Comment:
if `self.cfg.tensor_para_size == 1`, it will always true

Reply:
doesn't matter.